src/tests/unit/platform/test_alertsv7_api.py

```diff
@@ -568,6 +568,121 @@ def on_post(url, body, **kwargs):
+                                     "childproc_sha256": ["sha256_child"], "childproc_username": ["steven"],
+                                     "primary_event_id": ["123456"],
+                                     "parent_cmdline": ["/usr/bin/python"],
+                                     "parent_effective_reputation": ["PUP"],
+                                     "parent_guid": ["12345678"], "parent_name": ["python"],
+                                     "parent_sha256": ["sha256_parent"], "parent_username": ["steven"],
+                                     "process_cmdline": ["/usr/bin/python"],
+                                     "process_effective_reputation": ["PUP"],
+                                     "process_guid": ["12345678"], "process_name": ["IEXPLORE.EXE"],
+                                     "process_sha256": ["0123456789ABCDEF0123456789ABCDEF"],
+                                     "process_username": ["steven"],
+                                     "process_reputation": ["SUSPECT_MALWARE"],
+                                     "process_issuer": ["Microsoft"], "process_publisher": ["Microsoft"]},
+                        "sort": [{"field": "name", "order": "DESC"}]}
+        return {"results": [{"id": "S0L0", "org_key": "test", "threat_id": "B0RG",
+                             "workflow": {"status": "OPEN"}}], "num_found": 1}
+
+    cbcsdk_mock.mock_request('POST', "/api/alerts/v7/orgs/test/alerts/_search", on_post)
+    api = cbcsdk_mock.api
+
+    query = api.select(Alert).where("Blort").add_criteria("type", ["INTRUSION_DETECTION_SYSTEM"])\
+        .add_criteria("device_id", ["6023"]) \
+        .add_criteria("device_name", ["HAL"]).add_criteria("device_os", ["LINUX"]) \
+        .add_criteria("device_os_version", ["0.1.2"]).add_criteria("device_internal_ip", ["10.10.8.2"]) \
+        .add_criteria("device_username", ["JRN"]).add_criteria("id", ["S0L0"])\
+        .add_criteria("device_external_ip", ["10.10.10.55"]).add_criteria("device_uem_id", ["uemId"]) \
+        .add_criteria("severity", "6").add_criteria("device_policy_id", ["8675309"]) \
+        .add_criteria("device_policy", ["Strict"]).add_criteria("process_name", ["IEXPLORE.EXE"]) \
+        .add_criteria("process_sha256", ["0123456789ABCDEF0123456789ABCDEF"]) \
+        .add_criteria("process_reputation", ["SUSPECT_MALWARE"]) \
+        .add_criteria("netconn_remote_ip", ['10.29.99.1']).add_criteria("netconn_remote_domain", ['example.com']) \
+        .add_criteria("netconn_protocol", ['TCP']).add_criteria("netconn_local_port", ["12345"]) \
+        .add_criteria("netconn_remote_port", ["54321"]) \
+        .add_criteria("policy_applied", ["APPLIED"]).add_criteria("reason_code", ["ATTACK_VECTOR"]).add_criteria(
+        "run_state", ["RAN"]) \
+        .add_criteria("device_target_value", ["HIGH"]).add_criteria("threat_id", ["B0RG"]) \
+        .add_criteria("threat_name", ["dangerous"]).add_criteria("alert_notes_present", "True") \
```

Comment:
Instead of add_criteria("alert_notes_present", "True") needs to use  set_alert_notes_present("True")

Reply:
Line 352 still has this which is invalid
.add_criteria("alert_notes_present", "True")